### PR TITLE
py-awscli2: update to 2.22.1

### PR DESCRIPTION
**Pull request by ericbn**
###### Type(s)

- [ ] bugfix
- [X] enhancement
- [ ] security fix

###### Tested on

macOS 12.7.6 21H1320 x86_64
Command Line Tools 

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [X] checked your Portfile with `port lint --nitpick`?
- [X] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by dgilman** [comment]
Looking at the changelog, they dropped support for older macOS. The minimum version is now macOS 11. Can you update the portfile for that?

**Comment by ericbn** [comment]
Hi @dgilman. Thanks for reviewing this. PR updated.

**Review by dgilman — APPROVED** [review]

File: python/py-awscli2/Portfile [hunk 1]
```diff
@@ -6,11 +6,11 @@ PortGroup           select 1.0
 PortGroup           github 1.0
 
 name                py-awscli2
-github.setup        aws aws-cli 2.19.1
+github.setup        aws aws-cli 2.22.1
 revision            0
 
 categories          python devel
-platforms           {darwin any}
+platforms           {darwin >= 11}
```
Comment (reneeotten) [inline python/py-awscli2/Portfile:13]: If it supports macOS 11 and above this is bot correct. The Darwin version is not the same as the macOS version. 
See: https://trac.macports.org/wiki/XcodeVersionInfo

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: python/py-awscli2/Portfile (hunk 1 above)
Comment (ericbn) [reply python/py-awscli2/Portfile]: Oh, I was not aware of that, thank you. Just updated the PR.

**Pull request merged**